lib/birthday_bot.rb

```diff
@@ -11,17 +11,28 @@ def initialize()
 
   def start!
     birthdays = BirthdayReader.get_birthdays(@config.db_path)
-    today = Time.now 
 
-    puts "Checking who was born today (#{today.to_s})"
-    birthdays.each do |b|
-      if (b[3].to_i == today.month) && (b[4].to_i == today.day)
-        message = "#{@config.greeting_message} #{b[0]} #{b[1]}" 
-        HTTParty.post(@config.slack_url, body: { channel: @config.channel_name,
-                                                 username: @config.bot_name,
-                                                 text: message,
-                                                 icon_emoji: @config.bot_emoji }.to_json)
+    puts "Checking who was born today"
```

Comment:
Ah, you wanted it in the log. The way I handle my `cron`s are, I prepend `date` before my executing command. But I don't know if this works with Heroku's scheduler (does it?). Else I'll just put the date back in the log.